controller/customdomain_controller_test.go

```diff
@@ -253,6 +253,14 @@ func TestCustomDomainController(t *testing.T) {
 		objs = append(objs, cd)
 	}
 
+	infra := &configv1.Infrastructure{
```

Comment:
It doesn't look like you are testing anything new in this function. After running the reconcile, the timeout should be the new default timeout, but it doesn't look like you are testing that at all in this function.